deepquest/utils/bert_layer.py

```diff
@@ -0,0 +1,142 @@
+#! /usr/bin/env python
+# -*- coding: utf-8 -*-
+
+import logging
+
+import tensorflow as tf
+import tensorflow_hub as hub
+from tensorflow.keras import backend as K
+
+from keras.engine.base_layer import Layer
+
+
+logging.basicConfig(level=logging.INFO,
+                    format='[%(asctime)s] %(message)s', datefmt='%d/%m/%Y %H:%M:%S')
+logger = logging.getLogger(__name__)
```

Comment:
isn't this whole thing in layers.py now?

Reply:
yes, well spotted @drj11.

Reply:
so delete that and assuming it still runs we're good to go.

Reply:
Removed. Travis OK. Resolving.